tests/test_cli/test_config.py

```diff
@@ -676,8 +676,6 @@ def get_component_config_value(self) -> dict:
     def test_set_get_correct_path(self):
         """Test component value updated in agent config not in component config."""
         agent_config = self.load_agent_config()
-        assert not agent_config.component_configurations
```

Comment:
can you explain removal?

Reply:
Because I've updated the dummy aea config, `agent_config.component_configurations` won't be empty anymore